src/model_bias_analysis.py

```diff
@@ -255,6 +256,43 @@ def per_term_negative_rates(df, terms, model_families, threshold, text_col,
         records.append(term_record)
     return pd.DataFrame(records)
 
+### Summary metrics
+def diff_from_base(base_metrics, per_term_results, model_families, metric_column):
+    diffs = {}
+    for fams in model_families:
+        family_name = model_family_name(fams)
+        family_base_metrics = base_metrics[family_name]
+        metric_diff_sum = 0.0
+        for metric_list in per_term_results[family_name + metric_column]:
```

Comment:
I find the nested for loops confusing (despite coming up with the structure of these dataframes :-/).

I think this is iterating over the per-term rows, right? Maybe add a comment saying this loop is over each row, and that the 'metric_list' contains values of the metric for each copy of a model within the same model family.

Reply:
Done